apps/pipelines/nodes/nodes.py

```diff
@@ -488,3 +507,55 @@ def post_extraction_hook(self, output, state):
                 team=session.team,
                 data=output,
             )
+
+
+class AssistantNode(Passthrough):
```

Comment:
why does this inherit from `Passthrough`?

Reply:
That is an excellent question and observation. Copy pasta with a pinch of forgetfullness. I'll fix, thanks for pointing it out

Reply:
https://github.com/dimagi/open-chat-studio/pull/869/commits/05f72d601cbb53144d864d5dd7fbe26cc1da58c5